<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckOutSystem</class>
 <widget class="QWidget" name="CheckOutSystem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check out</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CheckOutSystem{border-image:url(:/new/prefix1/cos.png)}</string>
  </property>
  <widget class="QPushButton" name="checkOut">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>check out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 202, 126)</string>
      </property>
      <property name="text">
       <string>room number:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="roomNumber"/>
    </item>
    <item>
     <widget class="QPushButton" name="checkBtn">
      <property name="text">
       <string>check</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>72</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 202, 126)</string>
      </property>
      <property name="text">
       <string>client name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="nameLine"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 202, 126)</string>
      </property>
      <property name="text">
       <string>check in days:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="dayLine"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 202, 126)</string>
      </property>
      <property name="text">
       <string>other services:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="moneyLine_2"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 202, 126)</string>
      </property>
      <property name="text">
       <string>money should receive:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="moneyLine"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>CheckOutSystem</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
